docs/tutorials/intro.md

```diff
@@ -205,7 +204,7 @@ fn main() {
 }
 ```
 
-The ICU4X repository has test data checked in tree in `provider/testdata/data`, however it is recommended one generate data on their own as described in the [next section](#generating data). Under the hood, `icu_testdata` is simply loading this data.
+The ICU4X repository has test data checked in tree in `provider/testdata/data`, however it is recommended one generate data on their own as described in the [next section](#generating-data). Under the hood, `icu_testdata` is simply loading this data.
```

Comment:
_..._ has test data for some locales checked in at ... . To build data with your own selection of keys and locales, see the next section